### ISSUE TITLE
Add information on SolarSystem's license

### ISSUE THREAD

**Issue by AltNico** [elbanic/SolarSystem#1]
@elbanic First of all, thank you so much for building and publishing this piece of work. You've done a wonderful job and I really like the animation you created.

I'd like to extend your work, but to be able to do so, information on SolarSystem's license needs to be added to this project, by e.g. adding a `LICENSE` file. To me it doesn't matter which license you choose, as long as it permits othesr to modify and distribute your work. If you need help on this, feel free to reach out!

**Comment by elbanic** [comment]
I added `LICENSE ` file with commit https://github.com/elbanic/SolarSystem/commit/7290d6b0390987e60793c73d65df873a5ce26a1b.

**Comment by AltNico** [comment]
Thanks a lot for adding that license, @elbanic. I just published a blog post and the code of our fork and thought you might be interested in it. We extended your program by numerically calculating Earth's and Moon's orbits, as part of our Physics studies. We used your code to animate that solutions by giving OpenGL the calculated positions.

[3D Space Animations with Python - Nico's blog](https://nico.dorfbrunnen.eu/posts/2020/physics/)
